applications/tari_console_wallet/src/automation/prompt.rs

```diff
@@ -25,13 +25,14 @@ use std::{io, io::Write, str::FromStr};
 use tari_utilities::hex::{Hex, HexError};
 
 use crate::automation::error::CommandError;
-
+#[allow(dead_code)]
```

Comment:
I think either comment out the `mod prompt` in the parent module and add a comment saying that this is kept for future use _or_ add `#![allow(dead_code)]` to the top of this module with the same comment.